src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertSolutions.sql

```diff
@@ -532,16 +532,19 @@ TPP maintain close contact with staff at the unit throughout these phases to ens
           VALUES
           ('100000-001',	1, 1, 'F8D06518-1A20-4FBA-B369-AB583F9FA8C0', 2, 'GBP', @now, 99.99),
           ('100000-001',	1, 2, 'F8D06518-1A20-4FBA-B369-AB583F9FA8C0', 2, 'GBP', @now, NULL),
-          ('100001-001',	3, 1, '8BF9C2F9-2FD7-4A29-8406-3C6B7B2E5D65', 2, 'GBP', @now, 3.142),
+          ('100001-001',	3, 1, '8BF9C2F9-2FD7-4A29-8406-3C6B7B2E5D65', NULL, 'GBP', @now, 3.142),
           ('100002-001',	2, 1, 'F8D06518-1A20-4FBA-B369-AB583F9FA8C0', 1, 'GBP', @now, 4.85),
-          ('100004-001',	2, 1, '8BF9C2F9-2FD7-4A29-8406-3C6B7B2E5D65', 2, 'EUR', @now, 10101.65),
-          ('100005-001',	3, 1, '8BF9C2F9-2FD7-4A29-8406-3C6B7B2E5D65', 2, 'GBP', @now, 456),
+          ('100004-001',	2, 1, '8BF9C2F9-2FD7-4A29-8406-3C6B7B2E5D65', 1, 'EUR', @now, 10101.65),
+          ('100005-001',	3, 1, '8BF9C2F9-2FD7-4A29-8406-3C6B7B2E5D65', NULL, 'GBP', @now, 456),
           ('100006-001',	2, 1, '90119522-D381-4296-82EE-8FE630593B56', 1, 'GBP', @now, 7),
           ('99998-98', 1, 1, '8BF9C2F9-2FD7-4A29-8406-3C6B7B2E5D65', 2, 'GBP', @now,	30000),
-          ('99999-89', 1, 1, '8BF9C2F9-2FD7-4A29-8406-3C6B7B2E5D65', 1, 'GBP', @now,	500.49),
-          ('100002-001', 2, 2, 'F8D06518-1A20-4FBA-B369-AB583F9FA8C0', 2, 'GBP', @now, NULL),
+          ('99999-89', 1, 1, '8BF9C2F9-2FD7-4A29-8406-3C6B7B2E5D65', 2, 'GBP', @now,	500.49),
+          ('100002-001', 2, 2, 'F8D06518-1A20-4FBA-B369-AB583F9FA8C0', 1, 'GBP', @now, NULL),
           ('99998-98', 1, 2, '8BF9C2F9-2FD7-4A29-8406-3C6B7B2E5D65', 2, 'GBP', @now,	NULL),
-          ('99999-89', 1, 2, '8BF9C2F9-2FD7-4A29-8406-3C6B7B2E5D65', 1, 'GBP', @now,	NULL)
+          ('99999-89', 1, 2, '8BF9C2F9-2FD7-4A29-8406-3C6B7B2E5D65', 2, 'GBP', @now,	NULL),
+          ('100000-001', 3, 1, '774E5A1D-D15C-4A37-9990-81861BEAE42B', NULL, 'GBP', @now, 1001.010),
+          ('100003-001', 2, 1, 'D43C661A-0587-45E1-B315-5E5091D6E9D0', 1, 'EUR', @now, 19.987),
+          ('100007-001', 3, 1, '90119522-D381-4296-82EE-8FE630593B56', NULL, 'USD', @now, 0.15)
           ;
```

Comment:
Sorry, Mr Nit-pick here! Can you fix the formatting?

There are a few places where there are tabs instead of spaces and the terminating semicolon should be at the end of line 547.

Can you also remove the square brackets from the `INSERT` statement?

Ideally, I think these price inserts should be incorporated into the corresponding solution insert blocks. However, we can do that as part of a tech debt ticket.